ctapipe/io/simteleventsource.py

```diff
@@ -61,6 +61,15 @@
 }
 
 
+@enum.unique
+class MirrorClass(enum.Enum):
+    """Enum for the sim_telarray MIRROR_CLASS values"""
+    SINGLE_SEGMENTED_MIRROR = 0
+    SINGLE_SOLID_PARABOLIC_MIRROR = 1
```

Comment:
>  It might be better to go back to that rather than this number of mirrors splitting.

Agreed, but that would be another Enum and that has to include a hybrid between davies cotton and parabolic, which is used by the MSTs.

But that's really not connected to the code here.

Reply:
We can infer that information from the `focal_length`,  `dish_shape_length` and `parabolic_mirror` attributes though. 

LST:
```
FOCAL_Length = 2800                                                                                                                           
EFFECTIVE_FOCAL_Length = 2923.7 0 0 0 0 
PARABOLIC_DISH = 1                                                                                                                            
DISH_SHAPE_Length = 2800
```

MST:

```
FOCAL_Length = 1600                                                                                                                           
EFFECTIVE_FOCAL_Length = 1644.51 0 0 0 0            
PARABOLIC_DISH = 0                                                                                                                            
DISH_SHAPE_Length = 1920                                                                                                                      
MIRROR_FOCAL_Length = 1608.3 
```
```

Reply:
Sorry, I just noticed I misread this and thought it was part of the OpticsDesciption, but now I see it's just an internal attribute to SimTelEventSource. In that case it's ok - my comments were mostly that our OpticsDescription may need an overhaul, but that's another PR... 

I'm ok with this then.